### PR TITLE
fix(hasMacroNameInMend): default sets to true

### LINKED ISSUE #26: set hasMacroNameInMend to TRUE
As per the results of this popular [poll](https://www.linkedin.com/posts/allanbowe_sas-sasapps-sasjs-activity-6783413360781266945-1-7m) we should set the default for this setting to `true`

image

### PR DESCRIPTION
**Pull request by saadjutt01**
## Issue

Closes #26 

## Intent

Lint should has default value for `hasMacroNameInMend` to be `true`

## Implementation

- Updated `getLintConfig.ts` and `.sasjslint` to have true value for `hasMacroNameInMend`
- Updated specs

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
- [ ] Any new functionality has been unit tested.
- [x] All unit tests are passing (`npm test`).
- [ ] All CI checks are green.
- [ ] sasjslint-schema.json is updated with any new / changed functionality
- [ ] JSDoc comments have been added or updated.
- [x] Reviewer is assigned.

### CODE REVIEW

File: src/lint/lintFile.spec.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ describe('lintFile', () => {
       path.join(__dirname, '..', 'Example File.sas')
     )
 
-    expect(results.length).toEqual(8)
+    expect(results.length).toEqual(9)
```
Comment (YuryShkoda) [inline src/lint/lintFile.spec.ts:11]: we shouldn't use the exact numbers

File: src/lint/lintFolder.spec.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ describe('lintFolder', () => {
     const diagnostics = results.get(
       path.join(__dirname, '..', 'Example File.sas')
     )!
-    expect(diagnostics.length).toEqual(8)
+    expect(diagnostics.length).toEqual(9)
```
Comment (YuryShkoda) [inline src/lint/lintFolder.spec.ts:13]: we shouldn't use the exact numbers

File: src/lint/lintProject.spec.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ describe('lintProject', () => {
     const diagnostics = results.get(
       path.join(__dirname, '..', 'Example File.sas')
     )!
-    expect(diagnostics.length).toEqual(8)
+    expect(diagnostics.length).toEqual(9)
```
Comment (YuryShkoda) [inline src/lint/lintProject.spec.ts:18]: we shouldn't use the exact numbers

File: src/utils/getLintConfig.spec.ts [hunk 4]
```diff
@@ -17,7 +17,7 @@ describe('getLintConfig', () => {
     const config = await getLintConfig()
 
     expect(config).toBeInstanceOf(LintConfig)
-    expect(config.fileLintRules.length).toEqual(3)
+    expect(config.fileLintRules.length).toEqual(4)
```
Comment (YuryShkoda) [inline src/utils/getLintConfig.spec.ts:20]: we shouldn't use the exact numbers

**Comment by krishna-acondy** [comment]
We should probably replace assertions based on numbers with assertions based on `containEqual` for arrays. I'll make that change in a subsequent PR.

**Review by krishna-acondy — APPROVED** [review]

**Pull request merged**